src/ExadelMentorship.IntegrationTests/Features/WeatherTest.cs

```diff
@@ -13,7 +13,7 @@ public class WeatherTest
         public async Task GetTemperatureByCityName_WhenCityNameIsCorrect_ReturnsTemperature()
         {
             //Arrange
-            var weather = DI.Resolve<IWeather>();
+            var weather = DI.Resolve<ICurrentWeatherCommand>();
```

Comment:
and after removing `ICurrentWeatherCommand`, this test will focus testing the handler

Reply:
ok